docs/community.html

```diff
@@ -0,0 +1,231 @@
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+      </ul>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="developers.html">Developers</a>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="community.html">Community</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="https://lists.ovirt.org/archives/">Forum</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="blog.html">Blog</a></li>
+  </ul>
+
+
+    </nav>
+  </header>
+  
+
+  <main class="ov-main">
+   <section class="ov-masthead--community ov-masthead">
+      <h1 class="ov-heading ov-heading--xl">Get Involved</h1>
+      <h2 class="ov-heading ov-heading--lg">oVirt is a community driven virtualization project and people just like you are making it happen.</h2>
+    </section>
+
+<div class="ov-site-section">
+   <section class="ov-community-subsection">
+      <h2 class="ov-heading ov-heading--lg">Community</h2>
+        <p class="ov-content-styles"> oVirt is a community project, and we welcome contributions from everyone! If you’d like to write code, report bugs, contribute designs, or enhance the documentation, we would love your help!
+        </p>    
+        <p class="ov-content-styles">
+        There are a few ways to engage with the oVirt Community: 
+        </p>
+        <ul class="ov-content-styles">
+          <li>
+            <a href="https://ovirt.org/events/">Events</a>
+          </li>
+          <li>
+            <a href="#">Forums/ Mailing lists</a>
+          </li>
+          <li>
+            <a href="https://www.facebook.com/groups/ovirt.openvirtualization/">oVirt Facebook Group</a>
+          </li>
+          <li>
+            <a href="https://twitter.com/ovirt">Twitter</a>
+          </li>
+          <li>
+            IRC – #ovirt on irc.oftc.net
+          </li>
+        </ul>
+        <p class="ov-content-styles">
+          Before getting started, please read our <a href="https://ovirt.org/community/about/community-guidelines.html">community etiquette guidelines</a>.
+        </p>
+    </section>
+
+    <section class="ov-community-subsection">
+      <h2 class="ov-heading ov-heading--lg">Developers</h2>
+      <div>
+        <p class="ov-content-styles ">
+          If you’d like to contribute code to oVirt, visit the <a href="https://ovirt.org/develop/">Developer section</a> of the site for guidelines. All of our projects use git. Most are hosted at <a href="https://gerrit.ovirt.org/#/dashboard/self">gerrit.ovirt.org</a> , and the rest are hosted at <a href="https://github.com/oVirt/"> github.com/oVirt</a>. 
+        </p>  
+      </div>  
+    </section>
+
+    <section class="ov-community-subsection">
+      <h2 class="ov-heading ov-heading--lg">Translate</h2>
+      <div>
```

Comment:
I'd drop these `<div>`s they shouldn't be needed.

Reply:
Can do, I got rid of all of the divs and applied `ov-content-styles` class to all of the sections instead.